LICENSE

```diff
@@ -2,11 +2,11 @@ MIT License
 
 Copyright (c) 2016 Software Engineering Education - FOSS Resources
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
this should still be person, not changed to food ( issue occured when using assisted refactoring to change all person to food)

Reply:
Yes, this PR has some minor errors within some of the doc files etc. Will be fixed, but wanted more to show the refactoring as a whole and gather input on that. Still, thanks for pointing out, will look to fix the nits soon.